impf/decorators.py

```diff
@@ -83,6 +87,7 @@ def func(self, *args, **kwargs):
             else:
                 sleep(10)
             if not self.keep_browser: self.driver.close()
+            raise WorkflowException('Unexpected exception was raised!')
```

Comment:
Can we not just return? That should end the `control_`-functions. Having a `try-except` block in the `main.py` doesn't seem so nice, especially if it's not even logged. A lot of potential for "hidden errors" imo

Reply:
See my comment above why return will not end the execution of the "outer" control_main (or whatever comes before in the stack)